crypto/nextgen_crypto/src/ed25519.rs

```diff
@@ -498,6 +505,16 @@ pub mod compat {
             .no_shrink()
     }
 
+    /// Generates a well-known keypair `(Ed25519PrivateKey, Ed25519PublicKey)` for special use
+    /// in the genesis block.
+    pub fn generate_genesis_keypair() -> (Ed25519PrivateKey, Ed25519PublicKey) {
```

Comment:
Yep. Please extend the comment above indicating the expected module(s) where this would be used (e.g. consensus), the relationship with `vm_genesis::GENESIS_KEYPAIR`. I want to equip the reader with enough tools to start learning about the distinct semantics of "genesis"